src/python/gudhi/weighted_rips_complex.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Raphaël Tinarrage and Yuichi Ike
+#
+# Copyright (C) 2020 Inria, Copyright (C) 2020 FUjitsu Laboratories Ltd.
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+from gudhi import SimplexTree
+
+class WeightedRipsComplex:
+    """
+    class to generate a weighted Rips complex 
+    from a distance matrix and filtration value
+    """
+    def __init__(self, 
+                distance_matrix=None, 
+                filtration_values=None,
```

Comment:
`filtration_value` seems a bit vague, edges have filtration values (computed from this and the distances) for instance. Maybe `weights` or `vertex_weights` or something like that?